### PR TITLE
refactor: replace lint to revive

### PR DESCRIPTION
**Pull request by appleboy**
## lint time

```
$ time make lint
real    0m7.953s
user    0m5.928s
sys     0m2.129s
```

## [revive](https://github.com/mgechev/revive) time

```
$ time make revive
revive -config config.toml -exclude=./vendor/... ./... || exit 1

real    0m5.179s
user    0m8.248s
sys     0m1.785s
```

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: config.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+ignoreGeneratedHeader = false
```
Comment (techknowlogick) [inline config.toml]: Is it possible to name this `revive_config.toml` just to prevent any confusion about what this file is for?

Comment (appleboy) [reply config.toml]: @techknowlogick OK, do it.

File: Makefile [hunk 2]
```diff
@@ -126,12 +126,12 @@ errcheck:
 	fi
 	errcheck $(PACKAGES)
 
-.PHONY: lint
-lint:
```
[the remark this answers is not in the document]
Comment (appleboy) [reply Makefile]: @sapk revert.

File: config.toml (hunk 1 above)
Comment (appleboy) [reply config.toml]: @lafriks @techknowlogick change file name to `.revive.toml`.

**Review by techknowlogick — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Pull request merged**